From 72cebad1a79fb9fa1eb0b9fe66dda012c6e9a007 Mon Sep 17 00:00:00 2001
From: Colin Huang <colin.huang2@amd.com>
Date: Tue, 20 Oct 2026 16:02:55 +0800
Subject: [PATCH] configuration: anacapa: update leakdetector pin/object name

PinName change:
  ChannelX_leakage_YYYY -> ChannelX_leakage
Object name change:
  Manifold_1 -> Chassis_Floor
  Manifold_2 -> Manifold
  CPU_DIMM_Cold_Plate -> CPU_Cold_Plate

Gerrit: https://gerrit.openbmc.org/c/openbmc/entity-manager/+/96053

Change-Id: If628110895e748d1537478da447c41acdac13730
Signed-off-by: Colin Huang <colin.huang2@amd.com>
---
 configurations/meta/anacapa/anacapa_scm.json | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/configurations/meta/anacapa/anacapa_scm.json b/configurations/meta/anacapa/anacapa_scm.json
index 534e12f..9008421 100644
--- a/configurations/meta/anacapa/anacapa_scm.json
+++ b/configurations/meta/anacapa/anacapa_scm.json
@@ -311,7 +311,7 @@
         {
             "Level": "Warning",
             "Name": "EAM3_Cold_Plate",
-            "PinName": "Channel0_leakage_EAM3",
+            "PinName": "Channel0_leakage",
             "Polarity": "High",
             "SubType": "LeakSensingCable",
             "Type": "GPIOLeakDetector"
@@ -319,23 +319,23 @@
         {
             "Level": "Warning",
             "Name": "EAM0_Cold_Plate",
-            "PinName": "Channel1_leakage_EAM0",
+            "PinName": "Channel1_leakage",
             "Polarity": "High",
             "SubType": "LeakSensingCable",
             "Type": "GPIOLeakDetector"
         },
         {
             "Level": "Critical",
-            "Name": "Manifold_1",
-            "PinName": "Channel2_leakage_Manifold1",
+            "Name": "Chassis_Floor",
+            "PinName": "Channel2_leakage",
             "Polarity": "High",
             "SubType": "LeakSensingCable",
             "Type": "GPIOLeakDetector"
         },
         {
             "Level": "Critical",
-            "Name": "Manifold_2",
-            "PinName": "Channel4_leakage_Manifold2",
+            "Name": "Manifold",
+            "PinName": "Channel4_leakage",
             "Polarity": "High",
             "SubType": "LeakSensingCable",
             "Type": "GPIOLeakDetector"
@@ -343,15 +343,15 @@
         {
             "Level": "Warning",
             "Name": "EAM1_Cold_Plate",
-            "PinName": "Channel5_leakage_EAM1",
+            "PinName": "Channel5_leakage",
             "Polarity": "High",
             "SubType": "LeakSensingCable",
             "Type": "GPIOLeakDetector"
         },
         {
             "Level": "Warning",
-            "Name": "CPU_DIMM_Cold_Plate",
-            "PinName": "Channel6_leakage_CPU_DIMM",
+            "Name": "CPU_Cold_Plate",
+            "PinName": "Channel6_leakage",
             "Polarity": "High",
             "SubType": "LeakSensingCable",
             "Type": "GPIOLeakDetector"
@@ -359,7 +359,7 @@
         {
             "Level": "Warning",
             "Name": "EAM2_Cold_Plate",
-            "PinName": "Channel7_leakage_EAM2",
+            "PinName": "Channel7_leakage",
             "Polarity": "High",
             "SubType": "LeakSensingCable",
             "Type": "GPIOLeakDetector"
